pkg/snmp/traps/listener.go

```diff
@@ -77,12 +80,18 @@ func (t *TrapListener) Stop() {
 }
 
 func (t *TrapListener) receiveTrap(p *gosnmp.SnmpPacket, u *net.UDPAddr) {
+	packet := &SnmpPacket{Content: p, Addr: u, Timestamp: time.Now().UnixMilli(), Namespace: t.config.Namespace}
+	tags := packet.getTags()
+
+	t.aggregator.Count("datadog.snmp_traps.received", 1, "", tags)
+
 	if err := validatePacket(p, t.config); err != nil {
 		log.Debugf("Invalid credentials from %s on listener %s, dropping traps", u.String(), t.config.Addr())
 		trapsPacketsAuthErrors.Add(1)
+		t.aggregator.Count("datadog.snmp_traps.unknown_community_string", 1, "", tags)
```

Comment:
```suggestion
		t.aggregator.Count("datadog.snmp_traps.invalid_packet", 1, "", append(tags, "reason: unknown_community_string"))
```

Should use a more generic metric name to be able to support other type of errors in the future?

Reply:
Makes sense, updating this!